spec/set.js

```diff
@@ -0,0 +1,130 @@
+import chai, { expect, assert } from 'chai'
+import chaiChange from 'chai-change'
+import sinon from 'sinon'
+import Set from '../src/set'
+
+chai.use(chaiChange)
+
+describe.only( 'Set', () => {
```

Comment:
can we remove the .only method from this describe statement before merging to master?